src/test/java/org/openelisglobal/DbUnitUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.openelisglobal;
+
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import org.dbunit.DBTestCase;
+import org.dbunit.PropertiesBasedJdbcDatabaseTester;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.xml.FlatXmlDataSetBuilder;
+
+public class DbUnitUtil extends DBTestCase {
```

Comment:
Why not extend `DataSourceBasedDBTestCase` which leverages a DataSource object for database connections. This is generally preferred over directly setting connection details as it allows for more flexibility and easier management of connections

Reply:
And then the error log you reported above, is it the same as what ci logs are reporting?

Reply:
For more, you can take a look at: https://www.baeldung.com/java-dbunit